### ISSUE TITLE
TAM / AM Maplist bug

### ISSUE THREAD

**Issue by hvownij** [UT2004-Coding-Projects/3SPNC#3]
When upgrading the server to a new version of 3spn there seems to be maplist issues.

Also there are maplist issues when trying to use TAM and AM on the same server.

ex: I can only set one maplist in arenamaster and that is all that works in the UT2004 mapvote list. You cannot load 2 different maplists. This maybe a known issue with other UT2004 mods. Have seen a few things online. Would be nice to see if we can tweak our mod to work better with the default mapvoting system or tweak mapvote :)

v-k-g commented on Nov 28, 2020
@hvownij
This sounds like two issues. Is this correct?

When (and only when) updating to a new version of 3spn, something happens to the map list. What specifically? Does it eventually resolve itself?
Cannot use different map lists between game types. Have we tried b/w TAM and CTF? Maybe just an issue b/w TAM and AM?

____

TLDR - If you have AM and TAM on the server only 1 maplist works for both gametypes.  You cannot have a AM maplist and a TAM maplist on the same server for some odd reason.  changing the map prefix does not solve the issue.

**Comment by hvownij** [comment]
On my todo list!

**Comment by hvownij** [comment]
should be resolved  have not tested yet.  will leave this issue open until its verified.
